webapp/api-doc/Upload.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'API | upload_file'
+description: upload a file to the server
+---
+
+# API - upload_file - Description
+This call is used to upload a file to the service-vm so that it can be used by the user to run  applications on any of the owned clusters.
+
+## Basic Parameters
+| **Description** | upload a file to the server | | **URL**         | /backend/user_files         | | **HTTP Method** | POST                        | | **Security**    | Basic Authentication        |
+
+### Headers
+
+Type          | Description          | Required | Default value | Example value
+------------- | -------------------- | -------- | ------------- | -------------------------------------------
+Authorization | authentication token | `Yes`    | None          | tJ3b3f32f23ceuqdoS_TH7m0d6yxmlWL1r2ralKcttY |
+
+### Body Parameters
+
+Name          | Description             | Required | Default value | Example value
+------------- | ----------------------- | -------- | ------------- | -------------
+'file'        | file to be uploaded     | `Yes`    | None          |               |
+'description' | description of the file | `No`     | None          |               |
+
+## Example
+Example of request
+
+```
+curl -X POST -H "Authorization:Token tJ3b39yg7mceuqdoS_TH7m0d6yxmlWLMDenalKcttR" -F "file=project
```

Comment:
- Change POST to PUT here and above in the Basic Parameters Table
- Maybe it's better to write this example `curl` in one line